### PR TITLE
Use tap

### PR DESCRIPTION
**Pull request by Uzlopak**
I really love tap.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,18 +24,12 @@
     "http",
     "header"
   ],
-  "files": [
-    "README.md",
-    "LICENSE",
-    "index.js",
-    "types/index.d.ts"
-  ],
```
Comment (Fdawgs) [inline package.json:30]: But why? Made this the lightest repo in the entire org 😂

Comment (Uzlopak) [reply package.json]: reverted

**Review by Fdawgs — APPROVED** [review]

**Comment by Uzlopak** [comment]
can we add this package to the fastify-libraries team?

**Comment by Fdawgs** [comment]
> can we add this package to the fastify-libraries team?

I'll add it and the csrf package to the team tomorrow, not near a computer today and it's a pain on a phone

**Pull request merged**